build-tools/create-packs/Directory.Build.targets

```diff
@@ -48,6 +48,9 @@
       <_GlobalProperties Include="-p:Configuration=$(Configuration)" />
       <_GlobalProperties Include="-p:IncludeSymbols=False" />
     </ItemGroup>
+    <PropertyGroup>
+      <_BinlogPrefix>-bl:$(XamarinAndroidSourcePath)bin/Build$(Configuration)/msbuild-$([System.DateTime]::Now.ToString('yyyyMMddThhmmss'))-</_BinlogPrefix>
+    </PropertyGroup>
```

Comment:
OK this log is the same format as `make`:

![image](https://github.com/user-attachments/assets/88a0c13c-5e6f-403b-8f85-3bc61fb17b70)